src/services/sdk.ts

```diff
@@ -125,7 +124,7 @@ const buildPublication = (batchData: BatchFileData): Publication => {
 const dispatchActivityContent = (
   dispatch: Dispatch,
   message: BroadcastAnnouncement,
-  activityContent: ActivityContent,
+  activityContent: ActivityContentNote | ActivityContentProfile,
```

Comment:
I'm not sure what type is expected; this is just to make it work, as ActivityContent went away.

Reply:
This is the correct type. We took out `ActivityContent` because we wanted to be more explicit about what was profile data and what was post data.